crds/go.mod

```diff
@@ -0,0 +1,16 @@
+module albertzhong.com/crds
```

Comment:
Are you still using the `crds` folder?

Reply:
I should've deleted the `crds` folder, my bad